jest-mongodb-config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  mongodbMemoryServerOptions: {
+    binary: {
+      version: process.env.MONGO_BINARY_VERSION || '3.6.12',
+      skipMD5: true,
```

Comment:
i think we should verify the authenticity of the binary, it shouldn't take that much time